demo/src/main/java/com/novoda/demo/MainActivity.java

```diff
@@ -17,27 +17,28 @@
     private static final String URI_VIDEO_MP4 = "http://yt-dash-mse-test.commondatastorage.googleapis.com/media/car-20120827-85.mp4";
 
     private Player player;
+    private PlayerView playerView;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         Log.setShowLogs(true);
         setContentView(R.layout.activity_main);
+        playerView = (PlayerView) findViewById(R.id.player_view);
+    }
 
-        player = new PlayerFactory(this, PrioritisedPlayers.prioritiseExoPlayer()).create();
+    @Override
+    protected void onStart() {
+        super.onStart();
+        player = new PlayerFactory(this, PrioritisedPlayers.prioritiseMediaPlayer()).create();
```

Comment:
I like the change but I wonder if it is safe to do the change. Given my lack of knowledge on this area...